src/Validation/Constraint/ValidAt.php

```diff
@@ -29,7 +29,7 @@ private function guardLeeway(?DateInterval $leeway): DateInterval
         }
 
         if ($leeway->invert === 1) {
-            throw new InvalidArgumentException('Leeway cannot be negative');
+            throw new InvalidArgument('Leeway cannot be negative');
```

Comment:
This should be something like `Lcobucci\JWT\Validation\Constraint\LeewayCannotBeNegative`